routes/flock.json

```diff
@@ -119,43 +119,282 @@
         "content": [
-          "This is GOOSE III pod, the one that we are building in the next few terms to get ready for Competition III."
+          "The primary way of controlling the operating speed of the Linear Induction Motor (LIM) is to use an inverter to adjust the AC output frequency to the LIM using the DC battery power supply. The controller is used to ensure a stable power output from the non-constant three-phase AC loead from the LIM, while also accounting for the end effects (or \"turbulence\") produced by the LIM."
+        ],
+        "parallax": "/images/flock/goose4/motor-and-inverter.png"
+      },
+      {
+        "title": "Software and Embedded",
+        "content": [
+          "Designed with modularity and minimalism as a focus, the Software and Embedded team incorporates redundant and inexpensive devices that work together to control the pod. Multiple Raspberry Pis are used to aggregate data, communicate with the Arduinos through the CAN bus system and perform other complex tasks."
         ]
       }
     ]
   },
   {
     "name": "Goose V",
-    "thumbnail": "/images/flock/goosex.png",
+    "thumbnail": "/images/flock/goose5.png",
     "url": "goose_v",
+    "desc": "UnLIMited potential.",
+    "banner": "/images/flock/goosex/torontorender.jpg",
+    "sections": [
+      {
+        "title": "Specifications",
+        "content": [
+          "GOOSE V is the result of refinining and optimizing GOOSE IV. We decided to expand upon the feasibility of Linear Induction Motors by incorporating it alongside a conservative, scalable and cost-effective subsystem design approach. More specifically, our team has gained tremendous insights from testing a small-scale LIM prototype, a high speed flywheel braking rig and a physial battery failure scenario.",
+          "The prototype is about 0.533m (21.68\") wide, 2.296m (7.5') long, 0.356m (14.76\") tall, around 82 kg in weight, and is designed to reach a safe and faster speed of 44m/s."
+        ]
+      },
+      {
+        "title": "Linear Induction Motor",
+        "content": [
+          "Like a Rotary Induction Motor but cut up on its centre axis and with the stator and rotor unrolled flat, a linear induction motor produces thrust through a linear force rather than through torque. This allows it to function on an unpowered conductive track, thereby saving costs in maintenance and per-unit track costs. Despite the disadvantage of end effects (or \"turbulence\") leading to lower speeds, the LIM offers contactless propulsion on a simpler track such as an aluminum one. Furthermore, the LIM allows for a centralized design that reduces the weight of the entire pod, and there are a variety of opportunities to scale up the LIM design and promises of better levitation technologies to use with the LIM. We use the DLIM (or Double-sided LIM) design, which involves 2 LIMs on either side of the I-beam track, to allow for a stronger magnetic field to be maintained."
```

Comment:
"Like a Rotary Induction Motor but cut up on its centre axis and with the stator and rotor unrolled flat"

is too wordy. Change it to 

"Similar to a Rotary Induction Motor, a linear induction motor produces thrust through a linear force."

Reply:
And change the a linear induction motor to a Linear Induction Motor

Reply:
Also, i noticed you copied a lot from Goose 4. Whatever changes i suggested to thos sections, please implement it here.